src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.Windows.cs

```diff
@@ -13,869 +13,614 @@
 namespace System.Net.Security
 {
     //
-    // The class maintains the state of the authentication process and the security context.
-    // It encapsulates security context and does the real work in authentication and
+    // The class does the real work in authentication and
     // user data encryption with NEGO SSPI package.
     //
     // This is part of the NegotiateStream PAL.
     //
-    internal class NegoState
+    internal partial class NegoState
     {
-        private const int ERROR_TRUST_FAILURE = 1790;   // Used to serialize protectionLevel or impersonationLevel mismatch error to the remote side.
-
-        private static readonly byte[] s_emptyMessage = new byte[0];
-        private static readonly AsyncCallback s_readCallback = new AsyncCallback(ReadCallback);
-        private static readonly AsyncCallback s_writeCallback = new AsyncCallback(WriteCallback);
-
-        private Stream _innerStream;
-        private bool _leaveStreamOpen;
+        internal IIdentity GetIdentity()
+        {
+            CheckThrow(true);
 
-        private Exception _exception;
+            IIdentity result = null;
+            string name = _context.IsServer ? _context.AssociatedName : _context.Spn;
+            string protocol = "NTLM";
 
-        private StreamFramer _framer;
-        private NTAuthentication _context;
+            protocol = _context.ProtocolName;
 
-        private int _nestedAuth;
+            if (_context.IsServer)
+            {
+                SecurityContextTokenHandle token = null;
+                try
+                {
+                    //token = _context.GetContextToken();
```

Comment:
This method seems significantly different after the move.  Before it called GetContextToken then returned a new WindowsIdentity.  Now it does a lot more work, and the call to GetContextToken is commented out.

Maybe this is just inlining a method that got removed in the refactor? If so, this commented out line of code should at least be deleted.